cypress_bc98/cypress/robots/minetApp/BuyPage.cy.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseEyes, BaseHands } from "../BaseRobot"; 
+import { Dependencies, RobotEyes, RobotHands } from "./Robot";
+
+const baseRobotEyes = new RobotEyes();
+const baseRobotHands = new RobotHands();
+
+export class BuyPageEyes extends BaseEyes{
+    AssertHeaders(){
+        baseRobotEyes.getByText("Checkout")
+        baseRobotEyes.getByText("Buy Crypto")
+
+
+    }
+     DataChangeAfterSelecting(){
+        baseRobotEyes.getByText("Ethereum wallet")
+         
+    }
+    AssertCheckoutPage(){
+        baseRobotEyes.getByText("Purchase is completed, please check your balance in your crypto wallet")
+        baseRobotEyes.getByText("BUY CRYPTO")
+    }
+    AssertWatchData(){
+        baseRobotEyes.getByText("USD Coin")
```

Comment:
iterate the loop for common action with different values

Reply:
can use iterate but testing many components and functions are made according the page flow.

Reply:
if the same action used for different values, we can iterate the loop for the action/ method with different values using array